### PR TITLE
GraphQL: get contract list

### PR DESCRIPTION
**Pull request by PavelVesnin**

### CODE REVIEW

File: apps/graphql/lib/graphql_web/resolvers/contract_resolver.ex [hunk 1]
```diff
@@ -1,9 +1,44 @@
...
+  def list_contracts(%{filter: filter, order_by: order_by} = args) do
+    Contract
+    |> filter(filter)
+    |> order_by(^order_by)
+    |> Connection.from_query(&PRMRepo.all/1, args)
+  end
+
+  def filter(query, [{:legal_entity_relation, value} | tail]) do
+    foreign_field =
+      case value do
```
Comment (kiurchv) [inline apps/graphql/lib/graphql_web/resolvers/contract_resolver.ex]: We can map enum values directly to atoms

File: apps/graphql/test/web/resolvers/contract_resolver_test.exs [hunk 2]
```diff
@@ -38,11 +54,172 @@ defmodule GraphQLWeb.ContractResolverTest do
...
+
+    test "filter by closed date interval", %{conn: conn} do
+      nhs()
+
+      today = Date.utc_today()
+
+      for start_date <- [today, Date.add(today, -30)], do: insert(:prm, :contract, %{start_date: start_date})
+
+      variables = %{
+        filter: %{startDate: Date.Interval.to_edtf(%{first: today, last: Date.add(today, 10)})}
```
Comment (kiurchv) [inline apps/graphql/test/web/resolvers/contract_resolver_test.exs]:
```suggestion
        filter: %{startDate: to_string(%Date.Interval{first: today, last: Date.add(today, 10)})}
```

File: apps/graphql/lib/graphql_web/resolvers/contract_resolver.ex (hunk 1 above)
Comment (PavelVesnin) [reply apps/graphql/lib/graphql_web/resolvers/contract_resolver.ex]: look closer please, ```"MERGED_FROM" -> :merged_from_id```

File: apps/graphql/test/web/resolvers/contract_resolver_test.exs [hunk 3]
```diff
@@ -38,11 +54,172 @@ defmodule GraphQLWeb.ContractResolverTest do
...
+
+    test "filter by open date interval", %{conn: conn} do
+      nhs()
+
+      today = Date.utc_today()
+
+      for start_date <- [today, Date.add(today, -30)], do: insert(:prm, :contract, %{start_date: start_date})
+
+      variables = %{
+        filter: %{startDate: Date.Interval.to_edtf(%{first: today, last: nil})}
```
Comment (kiurchv) [inline apps/graphql/test/web/resolvers/contract_resolver_test.exs]:
```suggestion
        filter: %{startDate: to_string(%Date.Interval{first: today, last: nil})}
```

**Pull request merged**